docs/src/getting_started.md

```diff
@@ -8,8 +8,8 @@ You can obtain the release build via the [download page](https://zed.dev/downloa
 
 ### Configure Zed
 
-Use `CMD + ,` to open your custom settings to set things like fonts, formatting settings, per-language settings and more. You can access the default configuration using the `Zed > Settings > Open Default Settings` menu item. See Configuring Zed for all available settings.
+Use `⌘` + `,` to open your custom settings to set things like fonts, formatting settings, per-language settings and more. You can access the default configuration using the `Zed > Settings > Open Default Settings` menu item. See Configuring Zed for all available settings.
 
 ### Set up your key bindings
 
-You can access the default key binding set using the `Zed > Settings > Open Default Key Bindings` menu item. Use `CMD + K`,`CMD + S` to open your custom keymap to add your own key bindings. See Key Bindings for more info.,
+You can access the default key binding set using the `Zed > Settings > Open Default Key Bindings` menu item. Use `⌘` + `K` + `S` to open your custom keymap to add your own key bindings. See Key Bindings for more info.
```

Comment:
Here we still need two sequences of keystrokes, since it's a chord:

```suggestion
You can access the default key binding set using the `Zed > Settings > Open Default Key Bindings` menu item. Use `⌘` + `K`, `⌘` + `S` to open your custom keymap to add your own key bindings. See Key Bindings for more info.
```

Reply:
Oh, you _are_ able to perform this in two distinct steps. `⌘` + `K` + `S` seems to be valid as well. I'm not used to using "chord" shortcuts. Thanks.